src/main/resources/application.yml

```diff
@@ -168,6 +168,22 @@ workflow:
     retry-backoff-coefficient: "\${DEBT_POSITION_ACTIVITY_CONFIG_RETRY_BACKOFF_COEFFICIENT:\${workflow.default.retryBackoffCoefficient}}"
     retry-maximum-attempts: "\${DEBT_POSITION_ACTIVITY_CONFIG_RETRY_MAXIMUM_ATTEMPTS:\${workflow.default.retryMaximumAttempts}}"
 
+  payments-reporting-pagopa-organization-fetch:
+    start-to-close-timeout-in-seconds: "\${PAYMENTS_REPORTING_PAGOPA_ORGANIZATION_FETCH _START_TO_CLOSE_TIMEOUT_IN_SECONDS:\${workflow.default.startToCloseTimeoutInSeconds}}"
+    retry-initial-interval-in-millis: "\${PAYMENTS_REPORTING_PAGOPA_ORGANIZATION_FETCH_RETRY_INITIAL_INTERVAL_IN_MILLIS:\${workflow.default.retryInitialIntervalInMillis}}"
+    retry-backoff-coefficient: "\${PAYMENTS_REPORTING_PAGOPA_ORGANIZATION_FETCH_RETRY_BACKOFF_COEFFICIENT:\${workflow.default.retryBackoffCoefficient}}"
+    retry-maximum-attempts: "\${PAYMENTS_REPORTING_PAGOPA_ORGANIZATION_FETCH_RETRY_MAXIMUM_ATTEMPTS:\${workflow.default.retryMaximumAttempts}}"
+
+  payments-reporting-pagopa-fetch:
+    start-to-close-timeout-in-seconds: "\${PAYMENTS_REPORTING_PAGOPA_FETCH_START_TO_CLOSE_TIMEOUT_IN_SECONDS:\${workflow.default.startToCloseTimeoutInSeconds}}"
+    retry-initial-interval-in-millis: "\${PAYMENTS_REPORTING_PAGOPA_FETCH_RETRY_INITIAL_INTERVAL_IN_MILLIS:\${workflow.default.retryInitialIntervalInMillis}}"
+    retry-backoff-coefficient: "\${PAYMENTS_REPORTING_PAGOPA_FETCH_RETRY_BACKOFF_COEFFICIENT:\${workflow.default.retryBackoffCoefficient}}"
+    retry-maximum-attempts: "\${PAYMENTS_REPORTING_PAGOPA_FETCH_RETRY_MAXIMUM_ATTEMPTS:\${workflow.default.retryMaximumAttempts}}"
+
+schedule:
+    # Duration of the schedule in hours
+    frequency-hour: "\${SCHEDULE_PAYMENTS_REPORTING_PAGOPA_FREQUENCY_HOUR:6}"
```

Comment:
what should let think that this property is for payments-reporting?
the previous structure was good, you have just to remove the proeprty name (or id)